platform/macos/doc_classes/EditorExportPlatformMacOS.xml

```diff
@@ -75,6 +75,13 @@
 		<member name="codesign/custom_options" type="PackedStringArray" setter="" getter="">
 			Array of the additional command line arguments passed to the code signing tool.
 		</member>
+		<member name="codesign/entitlements/additional" type="String" setter="" getter="">
+			Additional data added to the root [code]&lt;dict&gt;[/code] section of the [url=https://developer.apple.com/documentation/bundleresources/entitlements].entitlements[/url] file. The value should be an XML section with pairs of key-value elements, e.g.:
+				[codeblock lang=text]
+				&lt;key&gt;key_name&lt;/key&gt;
+				&lt;string&gt;value&lt;/string&gt;
+				[/codeblock]
++		</member>
```

Comment:
```suggestion
		</member>
```

Reply:
Lol.  I copied the literal "+" from the pipeline error message.  My bad 😅

Reply:
fixed